index.html

```diff
@@ -123,130 +122,93 @@ <h2>Wlkom Bij InVaria</h2>
-                    </div>
-                    <div class="col-md-4 mb-3">
-                      <img
-                        class="img-fluid"
-                        alt="Modern buildings with solar panels"
-                        src="./img/carousel-img-2.png"
-                      />
-                    </div>
-                    <div class="col-md-4 mb-3">
-                      <img
-                        class="img-fluid"
-                        alt="Modern buildings with solar panels"
-                        src="./img/carousel-img-3.png"
-                      />
-                    </div>
-                  </div>
-                </div>
-                <div class="carousel-item">
-                  <div class="row">
-                    <div class="col-md-4 mb-3">
-                      <img
-                        class="img-fluid"
-                        alt="Modern buildings with solar panels"
-                        src="./img/carousel-img-2.png"
-                      />
-                    </div>
-                    <div class="col-md-4 mb-3">
-                      <img
-                        class="img-fluid"
-                        alt="Modern buildings with solar panels"
-                        src="./img/carousel-img-3.png"
-                      />
-                    </div>
-                    <div class="col-md-4 mb-3">
-                      <img
-                        class="img-fluid"
-                        alt="Modern buildings with solar panels"
-                        src="./img/carousel-img-1.png"
-                      />
-                    </div>
-                  </div>
-                </div>
-                <div class="carousel-item">
-                  <div class="row">
-                    <div class="col-md-4 mb-3">
-                      <img
-                        class="img-fluid"
-                        alt="Modern buildings with solar panels"
-                        src="./img/carousel-img-3.png"
-                      />
-                    </div>
-                    <div class="col-md-4 mb-3">
-                      <img
-                        class="img-fluid"
-                        alt="Modern buildings with solar panels"
-                        src="./img/carousel-img-1.png"
-                      />
-                    </div>
-                    <div class="col-md-4 mb-3">
-                      <img
-                        class="img-fluid"
-                        alt="Modern buildings with solar panels"
-                        src="./img/carousel-img-2.png"
-                      />
-                    </div>
-                  </div>
-                </div>
-              </div>
+          <div
+            class="gallery js-flickity"
+            data-flickity-options='{ "freeScroll": true, "wrapAround": true }'
+          >
+            <div class="gallery-cell">
+              <img
+                src="./img/carousel-img-1.webp"
```

Comment:
Make sure you replace these relative paths on all places, I see mixed approaches at the moment.

```suggestion
                src="img/carousel-img-1.webp"
```